deltachat-ffi/deltachat.h

```diff
@@ -4167,29 +4169,34 @@ int64_t          dc_lot_get_timestamp     (const dc_lot_t* lot);
  */
 #define DC_MSG_VIDEOCHAT_INVITATION 70
 
+/**
+ * @}
+ */
+
 
 /**
  * @defgroup DC_SOCKET DC_SOCKET
  *
  * These constants configure socket security.
+ * To set socket security,
+ * use dc_set_config() with the keys mail_security and/or send_security.
+ * If these flags are not specified, #DC_SOCKET_AUTO is used.
  *
  * @addtogroup DC_SOCKET
  * @{
  */
 
-/**
- * @}
- */
-
-
 /**
  * Choose socket security automatically.
+ * Automatic configuration is disabled if another @ref DC_SOCKET flag is specified
```

Comment:
If server or port is specified, but `mail_security` or `send_security` is auto, autoconfiguration still tries both STARTTLS and SSL/TLS options.

Reply:
k, so, we better add a hint on auto-configure to dc_configure()